### ISSUE TITLE
Data

### ISSUE THREAD

**Issue by 761010952** [499683288/Easy-permeability-prediction-for-porous-media#1]
Your code is great and helpful to me. I have a little data question to ask. I see that each sample of yours has 256 slices, and there are 5000 data in the data, which makes me very confused. I would like to ask you about it, thank you！

**Comment by njumyq** [comment]
You're right, I didn't put the full data here. The 256 slices here really only represent a 3D porous media sample. For the exact data you can look at https://zenodo.org/records/7735364, and https://zenodo.org/records/10858994.

**Comment by 761010952** [comment]
> 你是对的，我没有把完整的数据放在这里。这里的 256 个切片实际上只代表 3D 多孔介质样品。有关确切数据，您可以查看 https://zenodo.org/records/7735364 和 https://zenodo.org/records/10858994。

Thank you very much. You have helped me a lot with this job

**Comment by 761010952** [comment]
> 你是对的，我没有把完整的数据放在这里。这里的 256 个切片实际上只代表 3D 多孔介质样品。有关确切数据，您可以查看 https://zenodo.org/records/7735364 和 https://zenodo.org/records/10858994。

Thank you so much. Your work has helped me a lot
